### PR TITLE
Product pages accordion

### PR DESCRIPTION
**Pull request by dfullaway**
Combined work on accordion and project pages. Now both pull from the same data collection. Added all 4 projects (with as much information as is available). Unsure if accordion should only have 3 projects, or all 4. Currently has 4, but can be changed to 3. Also modified navigation to allow going to the top level pages.

### CODE REVIEW

File: src/components/elements/AccordionSection.svelte [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+</script>
+
+{#if activeValue==sequence}
+
+<div class="h-fit sm:h-full grid grid-cols-1">
+    <div class="grid grid-cols-5 p-4 content-center items-center">
+        {#if mobileValue}
+            <div class="col-span-1 text-6xl me-6 text-center font-menobanner text-mcswf-gold py"> {sequence}</div>
+        {/if}
+        <div class="col-span-3 text-left pt-2 tracking-widest sm:text-center font-bold block text-xl md:text-2xl lg:text-4xl"> {title} </div>
```
Comment (wcrum) [inline src/components/elements/AccordionSection.svelte]: Weird centering, this needs to be `sm:text-left`.

**Review by wcrum — CHANGES_REQUESTED** [review]
Other then one small thing, everything looks good to me. I commited additional logos and images you should use to replace the temporary pink background. Make the background and address the suggested change please.

File: src/components/elements/AccordionSection.svelte (hunk 1 above)
Comment (dfullaway) [reply src/components/elements/AccordionSection.svelte]: Adjusted as requested.

**Comment by dfullaway** [comment]
Updated metric icons are black, but other metric icons are white.

**Pull request merged**